web/angular-wallet/src/locales/bg.json

```diff
@@ -473,6 +473,7 @@
   "info": "Информация",
   "info_merchant_message_required": "Сметката на получателя изисква от вас да предоставите специално форматирано съобщение.",
   "info_referenced_transaction_hash": "Тъй като използвате сделка с референтен хеш, 100 BURST се удържат докато сделката не се потвърди или изтече срока.",
+  "input_pin": "Input PIN",
```

Comment:
i18nManager? :grin:

Reply:
find/replace :( - does it support adding?